fe/fe-core/src/main/java/com/starrocks/connector/jdbc/PostgresSchemaResolver.java

```diff
@@ -129,6 +129,10 @@ public Type convertColumnType(int dataType, String typeName, int columnSize, int
                 break;
         }
 
+        if (typeName.equalsIgnoreCase("uuid")) {
+            return ScalarType.createVarcharType(36);
```

Comment:
Hi, Thanks for contribution.
I'm curious why you want to convert UUID to Varchar(36). In Iceberg we convert it to VARBINARY, and from https://www.postgresql.org/docs/current/datatype-uuid.html, I think the length should be 128-bit(16Bytes), could you confirm that?

Reply:
I noticed that Apache Doris [converts](https://github.com/apache/doris/blob/3.0.5-rc01/fe/fe-core/src/main/java/org/apache/doris/datasource/jdbc/client/JdbcPostgreSQLClient.java#L167) uuid to string.
With current commit it looks like this:

```
MySQL [(none)]> select id from dwh.public.districts limit 2;
+--------------------------------------+
| id                                   |
+--------------------------------------+
| 8c1c7663-4cf4-11e6-80b8-fd42e350b981 |
| 7fd3aee6-4cf4-11e6-80b8-fd42e350b981 |
+--------------------------------------+
2 rows in set (0.056 sec)
```

Reply:
so what will happen if you create a olap table in StarRocks with binary type and insert into xxx select xxx from pg_table?

Reply:
Or maybe you need tread UUID in StarRocks as VarChar that what u have done in this pr

Reply:
Consider the definition, we'd better always treat UUID as 128-bit binary, so users can have a consistent approach in different systems.

Reply:
Hi Thanks for the contribution! Amazing work!
Could you pls convert it into varbinary as the iceberg catalog? so the same type will be converted into same type in StarRocks and won't cause confusion. Does this make sense to you?